lib/safira_web/live/backoffice/event_live/teams_live/form_component.ex

```diff
@@ -0,0 +1,52 @@
+defmodule SafiraWeb.Live.Backoffice.EventLive.TeamsLive.FormComponent do
+  use SafiraWeb, :live_component
+
+  alias Safira.Teams
+  import SafiraWeb.Components.Forms
+
+  @impl true
+  def render(assigns) do
+    ~H"""
+    <div>
+      <.page title={@title}>
+        <.simple_form for={@form} id="team-form" phx-target={@myself} phx-submit="save">
+          <div class="w-full space-y-2">
+            <.field field={@form[:name]} name="team[name]" type="text" label="Team Name" required />
+          </div>
+          <:actions>
+            <.button phx-disable-with="Saving...">Save Team</.button>
+          </:actions>
+        </.simple_form>
+      </.page>
+    </div>
+    """
+  end
+
+  @impl true
+  def update(assigns, socket) do
+    {:ok, assign(socket, assigns)}
+  end
```

Comment:
Like here, for instance, you could add something along the lines of this:

```suggestion
  @impl true
  def update(%{team: team} = assigns, socket) do
    {:ok, socket
         |> assign(assigns)
         |> assign_new(:form, fn ->
       to_form(Team.change_team(team))
     end)}}
  end
```